ecl/hql/CMakeLists.txt

```diff
@@ -113,6 +113,8 @@ if (WIN32)
     )
 else()
     add_custom_command ( OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/hqlgram.cpp ${CMAKE_CURRENT_BINARY_DIR}/hqlgram.h
+        #delete output files before building to force rebuilds with errors to fail rather than continue unnoticed, using the existing output files.
+        COMMAND ${bisoncmdprefix} rm -f ${CMAKE_CURRENT_BINARY_DIR}/hqlgram.h ${CMAKE_CURRENT_BINARY_DIR}/hqlgram.cpp
```

Comment:
For non windows it expands to blank - I would remove it.  I think it is there  because on windows bison is a batch file.